backend/de.metas.swat/de.metas.swat.base/src/main/java/org/adempiere/inout/util/ShipmentScheduleQtyOnHandStorageHolder.java

```diff
@@ -0,0 +1,46 @@
+package org.adempiere.inout.util;
+
+import com.google.common.collect.ImmutableList;
+import de.metas.inoutcandidate.api.OlAndSched;
+import de.metas.inoutcandidate.model.I_M_ShipmentSchedule;
+import de.metas.material.cockpit.stock.StockDataQuery;
+import lombok.Builder;
+import lombok.NonNull;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Loads stock details which are relevant to given {@link I_M_ShipmentSchedule}s.
+ * Allows to change (in memory!) the qtyOnHand.
+ */
+@Builder
```

Comment:
move the `@Builder` to ctor... or because you have only one field, i would suggest to use `@AllArgsConstructor(staticName="of")`

Reply:
moved.